modules/veteran/app/controllers/veteran/v0/representatives_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Veteran
+  module V0
+    class RepresentativesController < ApplicationController
+      skip_before_action :set_tags_and_extra_content, raise: false
+      skip_before_action :authenticate
+
+      def search
+        rep = Veteran::Service::Representative.for_user(target)
+        if rep.present?
+          render json: { data: rep.as_json }
+        else
+          render json: { errors: [{ detail: 'Representative not found' }] }
```

Comment:
Still not a fan of returning 200 with an error. My vote is for empty array here

Reply:
@ATeal  this is is updated now.